src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 950px;
+  margin: 0 auto;
+  padding: 50px 0;
+
+  &__card-margin {
+    margin-right: 50px;
+  }
+  
+  &__card-margin:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
the distance between cards on the design includes borders, so real margin-right is 48px, you can also use :not() to reduce the code